RevenueCatUI/Templates/V2/ViewModelHelpers/ViewModelFactory.swift

```diff
@@ -404,6 +404,9 @@ struct ViewModelFactory {
                     pageStackViewModels: pageStackViewModels
                 )
             )
+        case .video:
+            assert(false, "Video component not supported yet")
+            throw CocoaError(.featureUnsupported)
```

Comment:
Even though odds are this won't ever get called, I think I'd like to not merge any asserts or throws into a version that gets released 😬 

In this case, I would probably still create an nearly empty `VideoComponentViewModel` for compile reasons. And then in a future PR go and build out that implementation 💪